### DISCUSSION TITLE
2 Errors Report

### DISCUSSION THREAD

**Discussion by waswosi** [aclap-dev/video-downloadhelper#673] [Help and Q&A]
### Your question

On this this site the filename is displayed as xxxx_resolution. But the file at the end has format xxxxresolution. Sub-bar is missing.
https://watchmdh.to/

On this site the 1080p (for example) version is displayed for the file but only the smallest version gets downloaded.
https://www.mylf.com

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: de
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "M:\\###Trailer 2024-05-22###",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 8
}

**Comment by GrampaWildWilly** [comment]
Quoting site names is not helpful.  Supplying URLs of specific web pages on which videos are hosted is much more helpful.  And by helpful, I mean helpful to you.  I believe mylf is a for-pay porn site, if I'm not mistaken.  So any URLs from that site would be impossible for anybody to investigate, not because it's porn, but because you need a paid membership to access it.  I'm not familiar with the other site so I hope it does not require even a free registration.  Let's have at least one URL of a specific page from that site.
